ontology/yaml/resources/LIGHTING/entity_types/LCM.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lighting group controllers. 
+
+LCM_SS:
+  description: "Standard lighting group control module."
```

Comment:
did you mean group here?  The other LCM does not mention group.

Reply:
yes this is what i meant. lighting control module, used for controlling groups of lights.